airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -174,64 +232,22 @@ def get_modified_files(
 )
 @click.option("--ci-job-key", envvar="CI_JOB_KEY", type=str)
 @click.option("--show-dagger-logs/--hide-dagger-logs", default=False, type=bool)
-@click.pass_context
-@track_command
+@click_track_command
+@click_merge_args_into_context_obj
+@click_append_to_context_object("is_ci", lambda ctx: not ctx.obj["is_local"])
```

Comment:
@alafanechere @erohmensing Added a new decorator to standardize appending to the context, and make it closer to argument/option decorators.

basically replaces
```
ctx.obj["is_local"] = value
```

Not super attached but felt useful

Reply:
❓ In the case that setting the parameter is more complicated than just setting a value, would you suggest adding a util method and referencing it here? or doing it in the method itself?

I would potentially be looking into making this accept a coroutine in addition to a callable or value, if it fixes my problems with trying to append something. But that doesn't have to be done here :)

Reply:
Yup! Would definately recommend a named function instead of a lambda.

Also would welcome a coroutine version! Ill look into updating it to allow that